### ISSUE TITLE
Memory leak in UserMedicineSettingsFragmentActivity.

### ISSUE THREAD

**Issue by free4murad** [systers/malaria-app-android#188]
There is some code in UserMedicineSettingsFragmentActivity which can cause a memory leak.
These  objects are not collected by a GC event and can cause low memory problem.

@yatna @chhavip 
Should I make a PR.

**Comment by yatna** [comment]
@free4murad Please specify which code and what objects.

**Comment by free4murad** [comment]
```
private static Button mDoneButton;
private static TextView timePickButton;
private static View v;
private static TimePicker tp;
public static Context mFragmentContext;
```

Defining views and context as static causes memory leak. 
These are not collected by GarbageCollection events and cause memory problems.

**Comment by yatna** [comment]
Good. @free4murad Are such things present in other activities too?

**Comment by free4murad** [comment]
@yatna 
yes i saw it in some other activities as well.

**Comment by anubhakushwaha** [comment]
@free4murad Any updates?

**Comment by free4murad** [comment]
@anubhakushwaha was Waiting for approval.
Will send a PR today.

**Comment by anubhakushwaha** [comment]
Closing.Fixed via #226
